lib/ui/cards/staff_info/staff_info_card.dart

```diff
@@ -36,11 +52,12 @@ class _StaffInfoCardState extends State<StaffInfoCard> {
     super.didChangeDependencies();
   }
 
-  final _url =
-      "https://mobile.ucsd.edu/replatform/v1/qa/webview/staff_info.html";
+  //String _url = "https://cwo-test.ucsd.edu/WebCards/staff_info_new.html";
```

Comment:
is this commented here on purpsoe? do they need to be removed?

Reply:
It needs to be removed, I'll update the PR with the changes. Thanks for catching that, I thought I got all the commented out code.